pyodide_build/pypabuild.py

```diff
@@ -264,10 +323,22 @@ def _create_symlink_dir(env: dict[str, str], build_dir: Path | None):
         yield symlink_dir
         return
 
-    # Running from "pyodide build". Put symlinks in a temporary directory.
-    # TODO: Add a debug option to save the symlinks.
-    with TemporaryDirectory() as symlink_dir_str:
-        yield Path(symlink_dir_str)
+    # TODO: FIXME: compiler wrappers are still ending up in a temporary
+    # directory, which breaks persistent builds. This is not ideal, but
+    # it is better than nothing so this is non-blocking for now. It has
+    # to be investigated further.
+
+    if no_isolation:
+        # For non-isolated builds, create a persistent directory in the current working directory
+        # or in a well-known location like ~/.pyodide/compiler_wrappers
+        symlink_dir = Path.cwd() / ".pyodide_compiler_wrappers"
+        symlink_dir.mkdir(exist_ok=True)
+        yield symlink_dir
```

Comment:
I don't think we need to branch out this part. I gues we can pass the persistent directory for both no-isolated and isolated build (as I commented in https://github.com/pyodide/pyodide-build/pull/152#issuecomment-2764406264)

Reply:
Thanks for the pointer. I addressed this in 8325e803d78cc3e3833edceeb652aee2e7974891.